.env.example

```diff
@@ -1,9 +1,5 @@
-FLOW_WALLET_PORT=3005
-FLOW_WALLET_JOB_STATUS_WEBHOOK=http://host.docker.internal:8080/flow_wallet_api/webhook
-FLOW_WALLET_SCRIPT_PATH_CREATE_ACCOUNT=/custom_account_setup.cdc
-
 FLOW_WALLET_ADMIN_ADDRESS=0xf8d6e0586b0a20c7
-FLOW_WALLET_ADMIN_PRIVATE_KEY=bc075f150b041e732625a60e94cb15ac1cafa8510c2a44d09d76ff5165df2b7b
+FLOW_WALLET_ADMIN_PRIVATE_KEY=91a22fbd87392b019fbe332c32695c14cf2ba5b6521476a8540228bdf1987068
```

Comment:
Did we intend to change this here? I feel like this needs to match the value in the electables smart contract repo?